packages/iov-bns/src/decodemsg.ts

```diff
@@ -231,7 +231,7 @@ function decodeRegisterAccountTx(
     name: ensure(msg.name, "name"),
     owner: encodeBnsAddress(prefix, ensure(msg.owner, "owner")),
     targets: ensure(msg.targets, "targets").map(decodeBlockchainAddress),
-    broker: encodeBnsAddress(prefix, ensure(msg.thirdPartyToken, "owner")) || undefined,
+    broker: encodeBnsAddress(prefix, ensure(msg.broker, "owner")) || undefined,
```

Comment:
Is it intentional that broker is "owner" in `decodeRegisterAccountTx` but "broker" in `decodeRegisterDomainTx`?.
I understand that `thirdPartyToken` also did this but just wanted to be sure.

Reply:
Great catch!!!  That's a bug.  I'll fix it.

Reply:
Fixed by https://github.com/iov-one/iov-core/pull/1419